packages/gcp/src/index.ts

```diff
@@ -78,6 +79,7 @@ export interface IPubSubMessagePreprocessor {
 
 export interface IGcsClient {
     putObject(spanContext: SpanContext, body: Buffer, key: string): Promise<void>;
+    putObjectAsStream(context: SpanContext, stream: Readable, key: string): Promise<void>;
```

Comment:
Mind keeping variable names consistant
```suggestion
    putObjectAsStream(spanContext: SpanContext, stream: Readable, key: string): Promise<void>;
```

Reply:
Fixed.